### ISSUE TITLE
[bug][folia-fork-kaiijumc]java.lang.UnsupportedOperationException: null

### ISSUE THREAD

**Issue by cgw2023** [RRS-9747/HeadDrop#20]
headdrop cannot enable when using folia fork kaijumc
error occur
```
[09:32:48 INFO]: [HeadDrop] HeadDrop 4.20 by RRS
[09:32:49 INFO]: [com.zaxxer.hikari.HikariDataSource] HikariPool-1 - Starting...
[09:32:49 INFO]: [com.zaxxer.hikari.pool.HikariPool] HikariPool-1 - Added connection org.sqlite.jdbc4.JDBC4Connection@6b04d3ea
[09:32:49 INFO]: [com.zaxxer.hikari.HikariDataSource] HikariPool-1 - Start completed.
[09:32:49 ERROR]: Error occurred while enabling HeadDrop v4.20 (Is it up to date?)
java.lang.UnsupportedOperationException: null
        at org.bukkit.craftbukkit.v1_20_R1.scheduler.CraftScheduler.handle(CraftScheduler.java:533) ~[kaiiju-1.20.1.jar:git-Kaiiju-"1f0e0e1"]
        at org.bukkit.craftbukkit.v1_20_R1.scheduler.CraftScheduler.runTaskTimerAsynchronously(CraftScheduler.java:257) ~[kaiiju-1.20.1.jar:git-Kaiiju-"1f0e0e1"]
        at org.bukkit.craftbukkit.v1_20_R1.scheduler.CraftScheduler.runTaskTimerAsynchronously(CraftScheduler.java:244) ~[kaiiju-1.20.1.jar:git-Kaiiju-"1f0e0e1"]
        at org.bukkit.scheduler.BukkitRunnable.runTaskTimerAsynchronously(BukkitRunnable.java:140) ~[kaiiju-api-1.20.1-R0.1-SNAPSHOT.jar:?]
        at me.rrs.headdrop.HeadDrop.onEnable(HeadDrop.java:115) ~[HeadDrop.jar:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:281) ~[kaiiju-api-1.20.1-R0.1-SNAPSHOT.jar:?]
        at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.enablePlugin(PaperPluginInstanceManager.java:189) ~[kaiiju-1.20.1.jar:git-Kaiiju-"1f0e0e1"]
        at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.enablePlugin(PaperPluginManagerImpl.java:104) ~[kaiiju-1.20.1.jar:git-Kaiiju-"1f0e0e1"]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:507) ~[kaiiju-api-1.20.1-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.v1_20_R1.CraftServer.enablePlugin(CraftServer.java:642) ~[kaiiju-1.20.1.jar:git-Kaiiju-"1f0e0e1"]
        at org.bukkit.craftbukkit.v1_20_R1.CraftServer.enablePlugins(CraftServer.java:553) ~[kaiiju-1.20.1.jar:git-Kaiiju-"1f0e0e1"]
        at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:679) ~[kaiiju-1.20.1.jar:git-Kaiiju-"1f0e0e1"]
        at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:463) ~[kaiiju-1.20.1.jar:git-Kaiiju-"1f0e0e1"]
        at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:324) ~[kaiiju-1.20.1.jar:git-Kaiiju-"1f0e0e1"]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1188) ~[kaiiju-1.20.1.jar:git-Kaiiju-"1f0e0e1"]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:346) ~[kaiiju-1.20.1.jar:git-Kaiiju-"1f0e0e1"]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]
[09:32:49 INFO]: [HeadDrop] Disabling HeadDrop v4.20
[09:32:49 INFO]: HeadDrop Disabled.
```

**Comment by YTiceice** [comment]
https://github.com/RRS-9747/HeadDrop/blob/main/src/main/java/me/rrs/headdrop/HeadDrop.java#L107
Maybe it's because it's judged by whether the server name contains Folia.

**Comment by RRS-9747** [comment]
> headdrop cannot enable when using folia fork kaijumc error occur
> 
> ```
> [09:32:48 INFO]: [HeadDrop] HeadDrop 4.20 by RRS
> [09:32:49 INFO]: [com.zaxxer.hikari.HikariDataSource] HikariPool-1 - Starting...
> [09:32:49 INFO]: [com.zaxxer.hikari.pool.HikariPool] HikariPool-1 - Added connection org.sqlite.jdbc4.JDBC4Connection@6b04d3ea
> [09:32:49 INFO]: [com.zaxxer.hikari.HikariDataSource] HikariPool-1 - Start completed.
> [09:32:49 ERROR]: Error occurred while enabling HeadDrop v4.20 (Is it up to date?)
> java.lang.UnsupportedOperationException: null
>         at org.bukkit.craftbukkit.v1_20_R1.scheduler.CraftScheduler.handle(CraftScheduler.java:533) ~[kaiiju-1.20.1.jar:git-Kaiiju-"1f0e0e1"]
>         at org.bukkit.craftbukkit.v1_20_R1.scheduler.CraftScheduler.runTaskTimerAsynchronously(CraftScheduler.java:257) ~[kaiiju-1.20.1.jar:git-Kaiiju-"1f0e0e1"]
>         at org.bukkit.craftbukkit.v1_20_R1.scheduler.CraftScheduler.runTaskTimerAsynchronously(CraftScheduler.java:244) ~[kaiiju-1.20.1.jar:git-Kaiiju-"1f0e0e1"]
>         at org.bukkit.scheduler.BukkitRunnable.runTaskTimerAsynchronously(BukkitRunnable.java:140) ~[kaiiju-api-1.20.1-R0.1-SNAPSHOT.jar:?]
>         at me.rrs.headdrop.HeadDrop.onEnable(HeadDrop.java:115) ~[HeadDrop.jar:?]
>         at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:281) ~[kaiiju-api-1.20.1-R0.1-SNAPSHOT.jar:?]
>         at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.enablePlugin(PaperPluginInstanceManager.java:189) ~[kaiiju-1.20.1.jar:git-Kaiiju-"1f0e0e1"]
>         at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.enablePlugin(PaperPluginManagerImpl.java:104) ~[kaiiju-1.20.1.jar:git-Kaiiju-"1f0e0e1"]
>         at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:507) ~[kaiiju-api-1.20.1-R0.1-SNAPSHOT.jar:?]
>         at org.bukkit.craftbukkit.v1_20_R1.CraftServer.enablePlugin(CraftServer.java:642) ~[kaiiju-1.20.1.jar:git-Kaiiju-"1f0e0e1"]
>         at org.bukkit.craftbukkit.v1_20_R1.CraftServer.enablePlugins(CraftServer.java:553) ~[kaiiju-1.20.1.jar:git-Kaiiju-"1f0e0e1"]
>         at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:679) ~[kaiiju-1.20.1.jar:git-Kaiiju-"1f0e0e1"]
>         at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:463) ~[kaiiju-1.20.1.jar:git-Kaiiju-"1f0e0e1"]
>         at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:324) ~[kaiiju-1.20.1.jar:git-Kaiiju-"1f0e0e1"]
>         at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1188) ~[kaiiju-1.20.1.jar:git-Kaiiju-"1f0e0e1"]
>         at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:346) ~[kaiiju-1.20.1.jar:git-Kaiiju-"1f0e0e1"]
>         at java.lang.Thread.run(Thread.java:833) ~[?:?]
> [09:32:49 INFO]: [HeadDrop] Disabling HeadDrop v4.20
> [09:32:49 INFO]: HeadDrop Disabled.
> ```

can you give me the download link of kaijumc?

**Comment by YTiceice** [comment]
https://github.com/KaiijuMC/Kaiiju

**Comment by RRS-9747** [comment]
Fixed on 4.21
